packages/core/src/app/customer/Customer.tsx

```diff
@@ -206,9 +206,7 @@ class Customer extends Component<CustomerProps & WithCheckoutCustomerProps & Ana
             onError={onUnhandledError}
           />;
 
-        const isLoadingGuestForm = isWalletButtonsOnTop ?
-            isContinuingAsGuest || isExecutingPaymentMethodCheckout :
-            isContinuingAsGuest || isInitializing || isExecutingPaymentMethodCheckout;
+        const isLoadingGuestForm = isContinuingAsGuest || isExecutingPaymentMethodCheckout;
```

Comment:
I think we still need the `isWalletButtonsOnTop` check.

Reply:
I don't think so, because we have the same condition for both cases

Reply:
Do we need to update tests as part of this?

Reply:
Updated PR with extra tests